### DISCUSSION TITLE
Does Kurtosis expose ports to the public internet?

### DISCUSSION THREAD

**Discussion by guisellaDes** [kurtosis-tech/kurtosis#1277] [Q&A] [answered]
Does Kurtosis expose ports to the public internet?

**Comment by leeederek** [comment, marked as answer]
Hi @guisellaDes! Kurtosis does not currently allow you to expose any ports in your enclave to the internet. Service ports in enclaves are automatically mapped to ports on your local machine. If you're looking for a way to dev or test things on a shared enclave for your team, check out: https://github.com/kurtosis-tech/kurtosis/discussions/1246.

Hope this answers your question!
